net/nts/nts.go

```diff
@@ -267,14 +255,16 @@ type ExtHdr struct {
 	Length uint16
 }
 
-func (h ExtHdr) pack(buf *bytes.Buffer) error {
-	err := binary.Write(buf, binary.BigEndian, h)
-	return err
+func (h ExtHdr) pack(buf *[]byte, pos int) (int, error) {
```

Comment:
I wanted to keep the signature of all pack function the same. But if you think that is unecessary I will remove the error here.

Reply:
It's okay to keep the error but then callers should also handle them. Otherwise, this will fail should we at some point decide to actually return an error value that is not nil.

Reply:
Done

Decided to remove the error afterall